### PR TITLE
Base image update @ 02-02-2022

### PR DESCRIPTION
**Pull request by priitlatt**
Base image update @ 02-02-2022

### CODE REVIEW

File: content/specs/versions-macos-xcode-13-3.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+- **17.0.2** JAVA_HOME: `/Library/Java/JavaVirtualMachines/zulu-17.jdk/Contents/Home`
+- **16.0.2** JAVA_HOME: `/Library/Java/JavaVirtualMachines/zulu-16.jdk/Contents/Home`
+- **11.0.14** (default) JAVA_HOME: `/Library/Java/JavaVirtualMachines/zulu-11.jdk/Contents/Home`
+- **1.8.0_322** JAVA_HOME: `/Library/Java/JavaVirtualMachines/zulu-8.jdk/Contents/Home`
+
+## Xcode 13.3 (13E5086k)
+
+This is the Xcode version used by default when you select `13.3` in build settings in the workflow 
+editor for Flutter apps or set Xcode version to `13.3` in your codemagic.yaml file.
```
Comment (helinanever) [inline content/specs/versions-macos-xcode-13-3.md:79]: 13.3 will become the edge version instead of the 13.2 (needs updating on both pages)

**Comment by helinanever** [comment]
Also there is a reference to the available Xcode versions in ../specs/machine-type/#xcode-version-update-policy which needs updating.

File: content/specs/versions-macos-xcode-13-3.md (hunk 1 above)
Comment (priitlatt) [reply content/specs/versions-macos-xcode-13-3.md]: Addressed in 26fefbe.

**Comment by priitlatt** [comment]
https://github.com/codemagic-ci-cd/codemagic-docs/pull/705#issuecomment-1027858016

_Xcode version update policy_ section in  `../specs/machine-type` was updated in 26fefbe.

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**